src/test/java/org/candlepin/subscriptions/tally/UsageCalculationTest.java

```diff
@@ -52,6 +53,21 @@ void testAddToTotal() {
     assertNullExcept(calculation, HardwareMeasurementType.TOTAL);
   }
 
+  @Test
+  void testAddToTotalShouldRoundDoubleValues() {
+    UsageCalculation calculation = new UsageCalculation(createUsageKey("Product"));
+    calculation.addToTotal(getCores(), 2.9);
+    calculation.addToTotal(getCores(), 3.0);
+    calculation.addToTotal(getCores(), 2.3);
+    calculation.addToTotal(getCores(), 1.4);
+    calculation.addToTotal(getCores(), 0.2);
+
+    // Using the above metrics, without rounding the doubles, we got 9.799999999999999 instead of
```

Comment:
I don't think the context of this comment is going to age well.  I like having an explanation though, so mabye maybe move to a comment on the method that says something like "Added this test case due to a bug we've seen with doubles being rounded" or a reference to the SWATCH-2009 card

Reply:
Good idea. I've just changed it.